src/sqlfluff/rules/L006.py

```diff
@@ -19,8 +19,9 @@
 class Rule_L006(BaseRule):
     """Operators should be surrounded by a single whitespace.
 
-    | **Anti-pattern**
-    | In this example, there is a space missing space between the operator and 'b'.
+    **Anti-pattern**
+
+    In this example, there is a space missing space between the operator and ``b``.
```

Comment:
Removing duplicate word.
```suggestion
    In this example, there is a space missing between the operator and ``b``.
```